r/factset.protobuf.stach.v2/man/factset.protobuf.stach.v2.Rd

```diff
@@ -0,0 +1,33 @@
+\name{factset.protobuf.stach.v2}
+\alias{factset.protobuf.stach.v2}
+
+\title{factset.protobuf.stach.v2}
+
+\description{
+The purpose of this library is to help in the serialization and de-serialization of the FactSet STACH v2 format represented using Google Protocol buffers. This library has the required FactSet STACH v2 Protobuf files and the helper serialization related methods are provided by RProtobuf library which is an internal dependency.
+}
+
+\section{Usage}{
+   stachFormatdata <- read( factset.protobuf.stach.v2.Package, input='path of your stach extension file' )
```

Comment:
lets remove 'extension' word from this line

Reply:
done